homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -27,12 +45,19 @@
 )
 
 
-class YaleConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+class YaleConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Yale integration."""
 
     VERSION = 1
+    CONNECTION_CLASS = CONN_CLASS_CLOUD_POLL
```

Comment:
This isn't used anymore, not sure why it was added?

```suggestion
```

Reply:
Not intentional. Was a copy/paste mistake. Removed now.